Api/Configuration.md

```diff
@@ -7,21 +7,27 @@ The ASP.NET Core configuration model centers on two main constructs:
 - `ConfigurationBuilder` - describes how to construct the final configuration representation for your app
 - `IConfiguration` - holds the configuration values themselves.
 
-If you want to know more about how to manage configurations during development, check out our [User secrets](../best-practices/user-secrets) section.
-
 ### Configuration providers
 
-ASP.NET Core uses configuration providers to load key-value pairs from a variety of sources. Applications can use many different configuration providers. You can load values from files, environment variables, command-line arguments, a database, or a remote service, or you could create your custom configuration provider.
+ASP.NET Core uses [configuration providers](https://learn.microsoft.com/en-us/dotnet/core/extensions/configuration-providers) to load key-value pairs from a variety of sources. Those configuration providers can load values from files (like `appsettings.json` or the user secrets file), environment variables, command-line arguments, a database, or a remote service. You can even create your custom configuration provider if none of the above satisfy your needs.
```

Comment:
Just a minor typo at the end of the section, it should say `... none of the above satisfies your needs` instead of the `... none of the above satisfy your needs`